### PR TITLE
Removed support phone number and added our support email

### PR DESCRIPTION
**Pull request by Vegemash**

### CODE REVIEW

File: protected/config/core/common.php [hunk 1]
```diff
@@ -386,6 +386,8 @@
             ),
         ),
         'admin_email' => '',
+        'helpdesk_email' => 'cera.support@fivium.com.au',
+        'helpdesk_phone' => '',
```
Comment (apigram) [inline protected/config/core/common.php]: Are we actually going to set the helpdesk phone number at some stage?

**Review by apigram — COMMENTED** [review]
Does this appear on the admin page as well? From my understanding, anything set here will always override what gets specified in the admin screens.

**Comment by Vegemash** [comment]
Good point, that seems like a bug to me. If a user changes the message through the front-end and nothing happens that's clearly a problem

File: protected/views/base/_footer.php [hunk 2]
```diff
@@ -31,12 +31,14 @@
...
-					<li><?php echo Yii::app()->params['helpdesk_email'] ?></li>
-				<?php } ?>
-				<?php if (Yii::app()->params['helpdesk_phone']) { ?>
+        <?php
+        $help_email = SettingInstallation::model()->find('`key`="helpdesk_email"')['value'];
+        $help_email .= $help_email !== null ?null:Yii::app()->params['helpdesk_email']; ?>
+        <li><?php echo  $help_email?></li>
+				<?php
+        $help_phone = SettingInstallation::model()->find('`key`="helpdesk_phone"')[value];
+        $help_phone .= $help_email !== null ?null: Yii::app()->params['helpdesk_phone'];?>
```
Comment (apigram) [inline protected/views/base/_footer.php]: This should be $help_phone. Also fix spacing here and consider inverting the ternary/converting to a coalesce.

File: protected/views/base/_footer.php [hunk 3]
```diff
@@ -31,12 +31,14 @@
 		<div class="panel">
 			<ul class="inline-list">
 				<li><strong>Need help?</strong></li>
-				<?php if (Yii::app()->params['helpdesk_email']) { ?>
-					<li><?php echo Yii::app()->params['helpdesk_email'] ?></li>
-				<?php } ?>
-				<?php if (Yii::app()->params['helpdesk_phone']) { ?>
+        <?php
+        $help_email = SettingInstallation::model()->find('`key`="helpdesk_email"')['value'];
+        $help_email .= $help_email !== null ?null:Yii::app()->params['helpdesk_email']; ?>
```
Comment (apigram) [inline protected/views/base/_footer.php]: You need to fix the spacing here. Shouldn't need to tell you where.

Also, could you invert the ternary condition here? Having a null as the middle condition of a ternary doesn't seem like a good standard to set. Alternatively, use the coalesce operator ?:

**Review by apigram — CHANGES_REQUESTED** [review]

File: protected/views/base/_footer.php (hunk 3 above)
Comment (Vegemash) [reply protected/views/base/_footer.php]: Coalesce doesn't work in this case, it puts a 1 in. I'll invert the ternary.

**Review by apigram — APPROVED** [review]

File: protected/views/base/_footer.php (hunk 2 above)
Comment (marshl) [reply protected/views/base/_footer.php]: Fun fact: The coalesce operator ?: is also known as the Elvis operator.

**Review by marshl — APPROVED** [review]

**Pull request merged**